lib/features/settings/settings_screen.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter/material.dart';
```

Comment:
break App locale setting to another PR.

Each PR should fix only 1 issue.

Reply:
To be honest, at first I thought this issue was related to changing the language of the application, so I did it first. Then when I looked at it specifically I realized I was wrong. Instead of deleting it, I added it to this PR. I will separate it to make sure the PR only fixes 1 issue. Thank you,